main.go

```diff
@@ -529,9 +530,36 @@ func main() {
 		}
 	}
 
+	var configFiles []string
+
+	if opts.targetKubeConfig != "" {
+		configFiles = append(configFiles, opts.targetKubeConfig)
+
+		if targetK8sConfig.TLSClientConfig.CertFile != "" {
+			configFiles = append(configFiles, targetK8sConfig.TLSClientConfig.CertFile)
+		}
+	}
+
+	hubCfg, err := clientcmd.BuildConfigFromFlags("", opts.hubConfigPath)
+	if err != nil {
+		log.Error(err, "Could not load hub config")
+	} else {
```

Comment:
you can remove `else`

Reply:
I use the `hubCfg` in the else block so I'm making sure I'm not referencing a nil pointer there.